pootle/apps/pootle_misc/checks.py

```diff
@@ -967,14 +970,30 @@ def run_given_filters(checker, unit, check_names=[]):
 
 
 def get_qualitychecks():
-    sc = ENChecker()
-    for filt in sc.defaultfilters:
-        if filt not in excluded_filters:
-            # don't use an empty string because of
-            # http://bugs.python.org/issue18190
-            getattr(sc, filt)(u'_', u'_')
-
-    return sc.categories
+    available_checks = {}
+
+    if settings.QUALITY_CHECKER:
+        checkers = import_func(settings.QUALITY_CHECKER)()
```

Comment:
As we checked on IRC, this has to be a list: `checkers = [import_func(settings.QUALITY_CHECKER)()]`